exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -79,7 +80,12 @@ func createDefaultConfig() config.Exporter {
 		Namespace:        "",
 		ExternalLabels:   map[string]string{},
 		TimeoutSettings:  exporterhelper.DefaultTimeoutSettings(),
-		RetrySettings:    exporterhelper.DefaultRetrySettings(),
+		RetrySettings: exporterhelper.RetrySettings{
+			Enabled:         true,
+			InitialInterval: 50 * time.Millisecond,
+			MaxInterval:     200 * time.Millisecond,
+			MaxElapsedTime:  5 * time.Second,
```

Comment:
This seems like a very short time for max retries with exponential backoff. Sure it is much much better than the previous, but a max of 5 seconds would mean the retry sequence would look like:

[50ms, 100ms, 200ms, 400ms, 3.2s]

without jitter in there; so it'll perhaps try at most 5 times, which is quite short for example if a network just changed. I'd even put it to like a minute or 2 minutes. Please feel free to ignore though, given that this is a considerable improvement over the prior, but just food for thought.

Reply:
I agree. We should pick something larger. I chose 1 minute.

Reply:
Awesome, thank you and LGTM!